eventOrganizer/src/main/java/com/event/organizer/api/controller/EventController.java

```diff
@@ -57,4 +60,36 @@ private Event getEvent(EventRequestDto eventRequestDto) {
         }
         return event;
     }
+
+    private List<Event> getAllEvents(List<EventRequestDto> eventRequestDtos) {

```

Comment:
Is this method used somewhere? Why does it accept argument? 
This should be callable via GET Request, doesn't need any arguments.

Reply:
Here you should just make a call to eventService and fetch from the database all events that have status "Pending"

Reply:
In the future in the UI we will use this method, so that we can show in the "Organizers panel" all pending events that he must either accept or reject

Reply:
Change "getAllEvents" to "getAllPendingEvents"